languages/python/sqlalchemy-oso/sqlalchemy_oso/auth.py

```diff
@@ -29,7 +29,20 @@ def register_models(oso: Oso, base_or_registry):
     """Register all models in registry (SQLAlchemy 1.4) or declarative base
     class (1.3 and 1.4) ``base_or_registry`` with Oso as classes."""
     for model in iterate_model_classes(base_or_registry):
-        oso.register_class(model)
+        if model in oso.host.types:
+            # skip models that were manually registered
```

Comment:
This makes it possible for users to specify their own class names by registering them manually.